draft-ietf-rats-corim.md

```diff
@@ -1207,6 +1207,242 @@ The following describes each member of the `concise-bom-tag` map.
   The `$$concise-bom-tag-extension` extension socket is empty in this
   specification.
 
+# CoRIM-based Evidence Verification
+
+The verification procedure is divided into three separate phases:
+
+* Appraisal Context initialisation
+* Evidence collection
+* Evidence appraisal
+
+At a few well-defined points in the procedure, the Verifier behaviour will
+depend on the specific CoRIM profile.
+Each CoRIM profile MUST provide a description of the expected Verifier behavior
+for each of those well-defined points.
+
+Note that what follows describes a simplified implementation.
```

Comment:
```suggestion
Note that what follows describes a simplified algorithm for verification.
```

Reply:
Note that what follows describes a simplified algorithm.

Reply:
@nedmsmith : Me and Andrew thinks, "standard algorithm is neat and is specified in the standard document, so works for us atleast!

Reply:
WFM

Reply:
standard algorithm it is